internal/server/receiver_handlers.go

```diff
@@ -118,18 +118,22 @@ func (s *ReceiverServer) handlePayload(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var evaluator func(context.Context, client.Object) (*bool, error)
+	resourceFilter := func(ctx context.Context, o client.Object) (*bool, error) {
+		accept := true
+		return &accept, nil
+	}
 	if receiver.Spec.ResourceFilter != "" {
-		evaluator, err = newResourceFilter(receiver.Spec.ResourceFilter, r)
+		resourceFilter, err = newResourceFilter(receiver.Spec.ResourceFilter, r)
 		if err != nil {
-			logger.Error(err, "unable to create CEL evaluator")
+			logger.Error(err, "unable to create resource filter")
 			w.WriteHeader(http.StatusInternalServerError)
+			return
```

Comment:
This `return` is the main point of this PR, the rest is just fixing some style issues.